python/lsst/analysis/tools/__init__.py

```diff
@@ -21,19 +21,20 @@
 r"""This packages contains various analysis tools and PipelineTasks which
 run them.
 
-There are two types of primary tools within this package; `AnalysisMetrics
-and `AnalysisPlots`, both of which are specialized subclasses of `AnalysisTool`.
+There are two primary types of tools within this package: `AnalysisMetrics` and
+`AnalysisPlots`, both of which are specialized subclasses of `AnalysisTool`.
```

Comment:
There is `AnalysisTool` now, so this should be rewritten.

Reply:
Changed to (and merged with the next paragraph):
> The primary tool in this package is the `AnalysisTool` class.
> An `AnalysisTool` object has three stages: prep, process, and produce.
> Creating a new `AnalysisTool` involves choosing (through
> configuration) an `AnalysisAction` to run for each of those stages.